modules/tinymce/src/core/main/ts/autocomplete/AutocompleteContext.ts

```diff
@@ -37,10 +37,15 @@ const findStart = (dom: DOMUtils, initRange: Range, ch: string, minChars: number
     return Optional.none();
   }
 
-  const findTriggerChIndex = (element: Text, offset: number, text: string) =>
+  const buf: Array<{text: string; offset: number}> = [];
+
+  const findTriggerChIndex = (element: Text, offset: number, text: string) => {
     // Stop searching by just returning the current offset if whitespace was found (eg Optional.none())
     // and we'll handle the final checks below instead
```

Comment:
This comment should be below the buffer logic, as it applies to the `findChar(text, offset, ch).getOr(offset);` logic.